src/components/flame-graph/MaybeFlameGraph.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import * as React from 'react';
+import explicitConnect from '../../utils/connect';
+import { getInvertCallstack } from '../../reducers/url-state';
+import { changeInvertCallstack } from '../../actions/profile-view';
+import FlameGraph from './FlameGraph';
+
+import type {
+  ExplicitConnectOptions,
+  ConnectedProps,
+} from '../../utils/connect';
+
+require('./MaybeFlameGraph.css');
+
+type StateProps = {|
+  +invertCallstack: boolean,
+|};
+type DispatchProps = {|
+  +changeInvertCallstack: typeof changeInvertCallstack,
+|};
+type Props = ConnectedProps<{||}, StateProps, DispatchProps>;
+
+class MaybeFlameGraph extends React.PureComponent<Props> {
+  _onSwithToNormalCallstackClick = () => {
```

Comment:
Small typo here: `_onSwithToNormalCallstackClick` should be `_onSwitchToNormalCallstackClick`

Reply:
Ah, thanks!